### ISSUE TITLE
Implementing the opponent

### ISSUE THREAD

**Issue by Atrixsanna** [Isoldhe/Battleship#9]
Perhaps this should be a bot at first, doing random placement and random selection of targets?
Other thoughts include networking or turn-based on the same pc.

My preference for the actual game would be networking, but a bot for single player should be available as well (and this should be great for testing purposes too)

**Comment by Isoldhe** [comment]
Yeah, I prefer a multiplayer feature as well, but let's do the bot first and once that works for single player, we can implement a multiplayer.
